pkg/services/sqlstore/org_users_test.go

```diff
@@ -61,7 +62,7 @@ func TestSQLStore_GetOrgUsers(t *testing.T) {
 	}
 
 	store := InitTestDB(t)
-	store.Cfg.FeatureToggles = map[string]bool{"accesscontrol": true}
+	store.Cfg.IsFeatureToggleEnabled = featuremgmt.WithToggles("accesscontrol").IsEnabled
```

Comment:
we repeat `"accesscontrol"` a fair bit, should we maybe add that as a FLAG value?

Reply:
done